doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,20 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
+
+The following metadata are not included in notifications by default, the `metadata` field (within
+`notification` field) in subscription is used to do so.
+
```

Comment:
The `metadata` field (within the `notification` field) can be used to select a set of special attribute metadata included by notifications and aimed at providing more information about the nature of the event that triggered the notification. These metadata are: